megamek/src/megamek/common/Board.java

```diff
@@ -1023,6 +1024,14 @@ public void load(InputStream is) {
         load(is, null, false);
     }
 
+    public void load(String boardString) {
+        try (InputStream is = new ByteArrayInputStream(boardString.getBytes(StandardCharsets.UTF_8))) {
+            load(is, null, false);
+        } catch (IOException ex) {
+            logger.error(ex, "Error loading string to build board - {}", ex.getMessage());
+        }
+    }
```

Comment:
Allows me to load boards from hardcoded strings, great to make mock boards very quickly for tests